lib/shims/eq/R2015b/newline.m

```diff
@@ -0,0 +1,9 @@
+function result = newline
+
+% NEWLINE Create newline character
+% NEWLINE Introduced in R2016b
+% https://www.mathworks.com/help/matlab/ref/newline.html
+
+result = sprintf('\n');
+
+end  % function newline
```

Comment:
No "% function <blah>" trailing comments please.

Reply:
ok, I will change to honor this style preference, and get this into the open PR